codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/Rust.kt

```diff
@@ -86,6 +86,15 @@ object TestWorkspace {
     }
     private val subprojects = mutableListOf<String>()
 
+    private val cargoLock: File by lazy {
+        var curFile = File(this.javaClass.protectionDomain.codeSource.location.path)
+        while (!curFile.endsWith("smithy-rs")) {
```

Comment:
This breaks for me locally because I don't have the repo checked out under a directory named `smithy-rs`.